lib/modules/dosomething/dosomething_image/dosomething_image.install

```diff
@@ -18,3 +18,10 @@ function dosomething_image_schema() {
 function dosomething_image_update_7001(&$sandbox) {
   field_delete_field('field_video_poster');
 }
+
+/**
+ * Deletes field_video_id.
+ */
+function dosomething_image_update_7002(&$sandbox) {
+  field_delete_instance('field_video_id');
```

Comment:
This doesn't work!  https://api.drupal.org/comment/52613#comment-52613

Please pull down new stage DB and run `drush updb` then `drush -y fr dosomething_image` to test after fixing.